### DISCUSSION TITLE
Use cases for Live Sync

### DISCUSSION THREAD

**Discussion by Vyeche** [vrtmrz/obsidian-livesync#110] [Q&A]
Hi, 

What are the use cases of Live Sync?

Should it be used on multiple devices?

I read the message about the second device needing to be an empty vault. Does that mean I have to empty the vault each time I want to sync? 

Is there a guide on successfully implementing syncing between different devices and preventing corruption between devices? 
- What settings should I use?

For me, I noticed that when I rename files and move things around in the vault, the other devices get messed up—especially when a lot of changes take place.

Please advice.

Thanks & kind regards, 
Vye

**Comment by vrtmrz** [comment]
Sorry for being not clear! The vault is recommended to be empty only at first replication.

And we can configure synchronisation from preset instantly. It is the `🔧 Miscellaneous` on the setting dialogue.
There are two sorts of presets. Live and periodic with batch. If you are using obsidian on mobile devices, the period with batch is recommended.

**Reply by Vyeche** [reply]
How does the batch work together with the multiple devices?

**Reply by Vyeche** [reply]
When I setup the preset is it supposed to  toggle any of these switches? 

Will attach image soon.

**Reply by Vyeche** [reply]
![PXL_20220908_135432642](https://user-images.githubusercontent.com/972377/189146998-fe5eded6-5293-435b-84cf-1df73afa7167.jpg)
![PXL_20220908_135447715](https://user-images.githubusercontent.com/972377/189147004-eb842639-c9dc-4537-8b87-f9580bcb4001.jpg)
![PXL_20220908_135413738](https://user-images.githubusercontent.com/972377/189147006-b8341e26-84b7-4068-b0dd-829bb58e16e4.jpg)

**Reply by Vyeche** [reply]
Do any of these toggles need to switch after I set "Live and periodic with batch?"

**Reply by Vyeche** [reply]
Noticed that the presets disappears after I leave the screen.

![image](https://user-images.githubusercontent.com/972377/189155299-d1e97f1b-b163-4102-bcaf-1d2654edc9bf.png)
![image](https://user-images.githubusercontent.com/972377/189155323-51f68051-c41a-4065-91ff-7861c685416b.png)
![image](https://user-images.githubusercontent.com/972377/189155368-66ae7fb6-3a7b-473d-933d-bb5fbf8276d4.png)

Restarting the obsidian, I noticed the toggles coming, but present is blank.
![image](https://user-images.githubusercontent.com/972377/189156828-bc2fd3f0-4f2f-43af-ae99-0f261beb4d05.png)
![image](https://user-images.githubusercontent.com/972377/189156917-e233af2e-511c-4064-a371-c0492ba718a3.png)
![image](https://user-images.githubusercontent.com/972377/189157041-48e479ef-a25a-4f3b-bc3c-33f0c345c9cf.png)

**Reply by vrtmrz** [reply]
Disappearing preset is expected behaviour! It is just for applying the preset.
Instead of that, switches will be toggled on/off automatically.

I am getting increasingly convinced, The new documentation is needed and that the Setting pane needs to be reorganised. It gradually came to as like this.
